infra/base-images/base-runner/bad_build_check

```diff
@@ -113,6 +122,10 @@ function check_startup_crash {
     if [ $(egrep "target binary (crashed|terminated)" -c $FUZZER_OUTPUT) -eq 0 ]; then
       CHECK_PASSED=1
     fi
+  elif [[ "$FUZZING_ENGINE" = dataflow ]]; then
+    # TODO(https://github.com/google/oss-fuzz/issues/1632): add check for
+    # binaries compiled with dataflow engine when the interface becomes stable.
+    echo "Skipping check_startup_crash for dataflow engine build."
```

Comment:
I don't printing this line is helpful, same for previous case, please remove.

Reply:
Hmmm, you're right, that line will be visible in a case of failure, which shouldn't happen unless there is some actual check added. Oh, and I also need to do `CHECK_PASSED=1`. Done.